src/components/TextInput/TextInput.stories.tsx

```diff
@@ -0,0 +1,133 @@
+import React from "react";
```

Comment:
Is this file still being used for anything?  Most of the time when there's an MDX file in a component folder, it is being used as the component's sole story source.  In a few cases, some of the devs have their MDX files refer to the old TSX, but that doesn't seem to be what you're doing here.

Reply:
@kaplanmichael 
I took what was already in place with the Input component (.tsx)  and ran with that.  I knew that `.mdx` was the better way, but I balked and simply carried out what had already been started.  The `.mdx` file include here was my first attempt with that formatting, but I didn't get very far.  The `.tsx` file is fully functional for the story, but I would like to convert it to `.mdx`.  What do you think?  Do it now or come back to it after this component has been released?  I'm thinking to do it now.

Reply:
Sorry I missed this question you had asked me, but I do see you've moved forward with MDX and it looks great!